.github/actions/setup-environment/action.yml

```diff
@@ -44,19 +44,26 @@ runs:
     - name: Configure
       run: |
         ./eden config add default
-        ./eden config set default --key=eve.accel --value=false
+        if lscpu | grep -oEq "vmx|svm"; then
+          ./eden config set default --key=eve.accel --value=true
+        else
+          ./eden config set default --key=eve.accel --value=false
+        fi
         ./eden config set default --key=eve.tpm --value=${{ inputs.tpm_enabled }}
         ./eden config set default --key=eve.cpu --value=2
       shell: bash
       working-directory: "./eden"
     - name: Setup eve version
-      if: ${{ inputs.eve_image != '' }} and contains(${{ inputs.eve_image }}, ":" )
       run: |
         image=${{ inputs.eve_image }}
-        eve_pr_registry=$(echo "$image" |  cut -d ':' -f 1)
-        eve_pr=$(echo "$image" |  cut -d ':' -f 2 | cut -d "-" -f1)
-        ./eden config set default --key=eve.registry --value="$eve_pr_registry"
-        ./eden config set default --key=eve.tag --value="$eve_pr"
+        if [[ -n "$image" && "$input_string" == *:* ]]; then
```

Comment:
Curious, what is `input_string` here? From where it comes?

Reply:
That's from my bash ninja malpractices. I'll fix it tommorow. In should've been image